src/main/java/org/geysermc/generator/MappingsGenerator.java

```diff
@@ -832,6 +832,32 @@ public JsonObject getRemapBlock(BlockState state, String identifier) {
             bedrockIdentifier = "minecraft:moving_block";
         } else if (trimmedIdentifier.endsWith("note_block")) {
             bedrockIdentifier = "minecraft:noteblock";
+        } else if (trimmedIdentifier.equals("minecraft:cherry_sign")) {
```

Comment:
Can we use a map? Should make things easier for understanding when it's time to yeet the mappings.

Reply:
Yeah haha, I totally agree. I was getting disillusioned at this point